platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/GuestAuthenticationToken.java

```diff
@@ -27,33 +35,52 @@
   public static final String GUEST_TOKEN_VALUE_TYPE =
       BSTAuthenticationToken.BST_NS + BSTAuthenticationToken.TOKEN_VALUE_SEPARATOR + BST_GUEST_LN;
 
-  public GuestAuthenticationToken(String realm, String ip) {
-    super(new GuestPrincipal(ip), GUEST_CREDENTIALS, realm);
+  public GuestAuthenticationToken(String realm, String name) {
+    super(new GuestPrincipal(name), GUEST_CREDENTIALS, realm);
     setTokenValueType(BSTAuthenticationToken.BST_NS, BST_GUEST_LN);
     setTokenId(BST_GUEST_LN);
 
-    if (!StringUtils.isEmpty(ip)) {
-      SecurityLogger.audit("Guest token generated for IP address: " + ip);
+    if (!StringUtils.isEmpty(name)) {
+      SecurityLogger.audit("Guest token generated for IP address: " + name);
```

Comment:
This will still be the IP address. This is the token used internally, prior to the mapping of IP to internal ID that is shared out on the `GuestPrincipal`.